### DISCUSSION TITLE
Cancel pressure load at first element deletion

### DISCUSSION THREAD

**Discussion by bobimpopi** [OpenRadioss/OpenRadioss#2063] [Q&A]
Hello Everyone,

How hard would it be to modify /PLOAD to get a new Idel behavior that would cancel the whole load as soon as the first element associated to the segment is deleted? That would help modelling various piercing problems of inflated membrane I currently have. Any pointer on where I should look to do it myself if it's not too challenging?

Or maybe using a monitored volume would do, but I'm not sure about what happens when the elements are failing.

Thanks,
Etienne

**Comment by MaciekWronski** [comment]
Hi,

it's not straightforward since /PLOAD is based on surface defined by segments, not the elements.  However, if you define your surface using element groups, it could be possible using a new sensor checking the same element group for deleted elements and activating after the first element is off.
Maybe there are other solutions but this one should be one of the easiest to implement.
thx, kind regards
Maciek

Maciek Wronski
Director, Software Development

Altair | Nasdaq : ALTR
***@***.******@***.***> | altair.com<http://www.altair.com/>
Connect with us: LinkedIn<https://www.linkedin.com/company/altair-engineering/> | Twitter<https://twitter.com/altair_inc> | Facebook<https://www.facebook.com/altairengineering/>
#OnlyForward

________________________________
De : bobimpopi ***@***.***>
Envoyé : lundi 13 novembre 2023 09:01
À : OpenRadioss/OpenRadioss ***@***.***>
Cc : Subscribed ***@***.***>
Objet : [EXTERNAL] [OpenRadioss] Cancel pressure load at first element deletion (Discussion #2063)


CAUTION: This email originated from outside the organization. Do not click links or open attachments unless you recognize the sender and know the content is safe.

Hello Everyone,

How hard would it be to modify /PLOAD to get a new Idel behavior that would cancel the whole load as soon as the first element associated to the segment is deleted? That would help modelling various piercing problems of inflated membrane I currently have. Any pointer on where I should look to do it myself if it's not too challenging?

Or maybe using a monitored volume would do, but I'm not sure about what happens when the elements are failing.

Thanks,
Etienne

**Reply by bobimpopi** [reply]
Thank you Maciek for the pointer.

The sensor example where it gets activated based on Von Mises criterion is a bit complex as it requires a custom material, your idea of iterating over elements of a group seems easier to tackle. I'll have a look at the sensors code and give it a try. 

Regards,
Etienne
